test/fixtures/track/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>track</title>
+  <meta name="description" content="test that track creates a cookie with the expected expiry"></meta>
+  <script src="/dist/td.js"></script>
+</head>
+<body>
+
+<script type="text/javascript">
+  (function () {
+    function setBackground (color) {
+      document.body.style.background = color
+    }
+
+    function setStatus (text) {
+      document.getElementById('status').innerHTML = text
+    }
+
+    var td = new Treasure({
+      database: 'test_db_request',
+      writeKey: '91/96da3cfb876cc50724d0dddef670d95eea2a0018',
+      host: 'in.treasuredata.com',
+      storage: {
+        expires: 5
+      }
+    })
+
+    if (td.client.storage.expires !== 5) {
```

Comment:
I'd actually verify using raw js access to cookies rather than going deep into the td package where we could modify data ourselves to pass this test

Reply:
That is just our internal config, I was just confirming that we're storing what is passed in correctly. AFAIK it's not possible to inspect a cookie expiry from client JS